doc/source/notebooks/tailor/mixture_density_network.pct.py

```diff
@@ -230,7 +230,7 @@ def moon_data(N, noise):
 
 # %%
 Scipy().minimize(
-    tf.function(model.training_loss_closure(data)),
+    model.training_loss_closure(data),
```

Comment:
```suggestion
    model.training_loss_closure(data, jit=True),
```
if you want to be consistent